apps/api/src/domain/models/digital-text/entities/digital-text.entity.ts

```diff
@@ -338,6 +367,103 @@ export class DigitalText extends Resource {
         return this;
     }
 
+    @UpdateMethod()
+    importPages(pagesToImport: DigitalTextPageImport[]): ResultOrError<DigitalText> {
+        if (pagesToImport.length === 0) {
+            return new EmptyPageImportError();
+        }
+
+        if (this.hasPages()) {
+            return new CannotImportPagesToNonEmptyDigitalTextError(this.pages);
+        }
+
+        const pageUpdateResult = pagesToImport.map(
+            ({
+                pageIdentifier,
+                photographId,
+                audioAndTextContent,
+                // audioItemId,
+                // text,
+                // translation,
+                // languageCodeForText,
+                // languageCodeForTranslation,
+            }) => {
+                const originalLanguageImportItemSearchResult = audioAndTextContent.filter(
+                    ({ isOriginalText }) => isOriginalText
+                );
+
+                if (originalLanguageImportItemSearchResult.length > 1) {
+                    return new MultipleOriginalItemsInPageImportError();
+                }
+
+                if (originalLanguageImportItemSearchResult.length === 0) {
+                    return new MissingOriginalTextItemInPageImportError();
+                }
+
+                const {
+                    text: originalText,
+                    languageCode: originalLangaugeCode,
+                    // ~~audioItemId~~ we deal with this within the reduce loop below
+                } = originalLanguageImportItemSearchResult[0];
+
+                const pageWithOriginalTextOnly = new DigitalTextPage({
+                    identifier: pageIdentifier,
+                    content: buildMultilingualTextWithSingleItem(
+                        originalText,
+                        originalLangaugeCode
```

Comment:
(sp): `originalLangaugeCode` should be `originalLanguageCode`

Reply:
I fixed this and the other 25 occurrences of `langaugeCode`. Guess I need to watch my spelling on this one!